casper/src/main/scala/coop/rchain/casper/util/comm/DeployRuntime.scala

```diff
@@ -99,25 +102,27 @@ object DeployRuntime {
             println(s"Sending the following to Casper: ${d.term}")
           }
       response <- DeployService[F].deploy(d)
-      msg      = response.fold(processError(_).getMessage, "Response: " + _)
+      msg      = response.fold(_.mkString(System.lineSeparator()), "Response: " + _)
       _        <- Sync[F].delay(println(msg))
       _        <- Time[F].sleep(4.seconds)
     } yield ()
 
-  private def gracefulExit[F[_]: Monad: Sync, A](program: F[Either[Throwable, String]]): F[Unit] =
+  private def gracefulExit[F[_]: Monad: Sync, A](
+      program: F[Either[Seq[String], String]]
+  ): F[Unit] =
     for {
       result <- Sync[F].attempt(program)
-      _ <- result.joinRight match {
-            case Left(ex) =>
+      _ <- processError(result).joinRight match {
```

Comment:
@tymm No, this is not correct. See the `attempt` above. If we have `Left[Throwable, Either[Seq[String], A]]` this means that the whole request failed (f.e. because the node is unavailable) and we have no response (no service errors and no result). On the other hand, if we have `Right[Throwable, Either[Seq[String], A]]` then the request succeeded (no `Throwable` inside) and we can reduce the response to just `Either[Seq[String], A]`. I hope that my explanation is clear. :)

BTW, the `Left` unification happens here:
```scala
private def processError[A](error: Either[Throwable, A]): Either[Seq[String], A] =
  error.leftMap(_.toMessageList())
```